site/proxy.md

```diff
@@ -104,6 +104,15 @@ Multiple IP addresses and networks can be supplied in the `WEAVE_CIDR`
 variable by space-separating them, as in
 `WEAVE_CIDR="10.2.1.1/24 10.2.2.1/24"`.
 
+The docker NetworkSettings (including IP address, MacAddress, and
+IPPrefixLen), will still be returned by `docker inspect`. If you want
+`docker inspect` to return the weave NetworkSettings instead, then the
+proxy must be launced with the `--rewrite-inspect` flag. This will
+only substitute in the weave network settings when the container has a
+weave IP.
```

Comment:
Should we document what happens if the container has more than one weave IP address?

(it looks like we return the first one output by `ip addr show dev ethwe`)